compiler/rustc_mir_transform/src/copy_prop.rs

```diff
@@ -175,4 +175,11 @@ impl<'tcx> MutVisitor<'tcx> for Replacer<'_, 'tcx> {
             self.super_statement(stmt, loc);
         }
     }
+
+    fn visit_terminator(&mut self, terminator: &mut Terminator<'tcx>, location: Location) {
+        match &terminator.kind {
+            TerminatorKind::Call { .. } => {}
```

Comment:
What happens if we keep propagating into func and operands, except when it turns a `Move` into a `Copy` in `visit_operand`?

Reply:
Hm. I think I finally got this right, but the Copy argument in the `copy-prop/borrowed_local` test is no longer optimized?

Reply:
It appears things are fine and I am just a derp. I didn't bless.